articles/ros2_access_control_policies.md

```diff
@@ -99,10 +99,28 @@ Attributes:
 - **version**: declared version of schema version in use
   - Allows for advancing future revisions of the schema
 
+### `<contexts>` Tag
+
+Encapsulates a sequence of unique contexts.
+This method of nesting sequences allows for additional tags to be extended to the `<policy>` root.
+
+### `<context>` Tag
+
+Encapsulates a collection of profiles.
+This is specific to a context as determined by associative attributes.
+
+Attributes:
+- **path**: Fully qualified context path
```

Comment:
It's not a fully-qualified context name?

Reply:
Having the FQ path as an XML attribute in the context element would hint to SROS2 cli where to write the security artifacts to in the keystore. Or are you suggesting a hyphen?

Reply:
I think he's referring to context *name* vs *path*.
Which are the same here. This is equivalent to "Fully qualified context name"

@kyrofa Are you suggesting naming the attribute differently than the current "path"?

Reply:
IMO, given `/foo/bar/baz`, to me:
* "name" to means `baz`
* "namespace" means `/foo/bar`
* "path" is a non-colliding term for `/foo/bar/baz` , like as in path to a file

Reply:
I think the use of the word "path" here is correct, but it is inconsistent with terminology in the rest of ROS 2 so far. I think that's at least partially evident in @kyrofa's initial comment.

Reply:
Indeed, all I was saying was that, given other terminology, I expected this to be "name" instead of "path" (but the meaning is clear). Path makes me think path to a file, which feels a little odd, that's all. Elsewhere we call such things fully-qualified names. No strong feelings though.

Reply:
I'll just keep this as path to help disambiguate it with name or ns attributes in the profile tags.